### DISCUSSION TITLE
How do I find my node?

### DISCUSSION THREAD

**Discussion by SparkTomDog** [coder/code-server#5363] [Q&A]
> Entering the `node` command on my server shows

```bash
Welcome to Node.js v16.16.0.
```

> In `code-server bash` it shows command not found
```bash
bash: node: command not found
```

*switched to `root` but can't find*

**Comment by jsjoeio** [comment]
Interesting  🤔 How are you running code-server, how is it installed and what are the specs for your environment (OS, browser, etc.)?

What happens if you run `which node`? Also what does your `.bashrc` look like? I'm guessing it's not in your `PATH`

**Comment by SparkTomDog** [comment]
My server is `ubuntu` and I'm running with `root` and I have `nodejs` installed globally and it's working fine in the ssh client.
I use `docker` to host the project, but in the Terminal of code-server, it shows that the node command cannot be found

**Reply by jsjoeio** [reply]
So is code-server running inside a Docker container? I think you need Node installed there. Here is how I do it:

https://github.com/jsjoeio/dotfiles/blob/main/Dockerfile#L55-L63
